src/main/java/com/khoros/twitterapp/services/TwitterService.java

```diff
@@ -174,22 +191,36 @@ public Status createNewStatusObject(twitter4j.Status originalStatus) {
         newUser.setProfileImageUrl(originalStatus.getUser().get400x400ProfileImageURL());
 
         Status newStatus = new com.khoros.twitterapp.models.Status();
-        String url = new StringBuilder()
-                .append("https://twitter.com/")
-                .append(handle)
-                .append("/status/")
-                .append(originalStatus.getId())
-                .toString();
         newStatus.setMessage(originalStatus.getText());
         newStatus.setUser(newUser);
         newStatus.setCreatedAt(originalStatus.getCreatedAt());
-        newStatus.setPostUrl(url);
+        newStatus.setStatusID(originalStatus.getId());
```

Comment:
why the change to an existing api?

Reply:
I need to be able to access the status ID in order to set the inReplyTo for the reply. I would have had a statusID field from the beginning if I had known this functionality would be necessary later. Originally I was just trying to make it easier for the user of the api by supplying them with the tweet link, but the link can easily be built on their end as long as they have the statusID